charts/values.yaml

```diff
@@ -100,7 +100,18 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 
-
+# Configure network options
```

Comment:
Add a reference for network options configuration: https://kubernetes.io/docs/concepts/services-networking/network-policies/

Reply:
added.